src/Models/LoginSession.php

```diff
@@ -285,15 +294,24 @@ public function isCurrent(?HTTPRequest $request = null): bool
      */
     public static function getCurrentSessions(Member $member)
     {
-        $sessionLifetime = static::getSessionLifetime();
-        $maxAge = DBDatetime::now()->getTimestamp() - $sessionLifetime;
+        $maxAge = self::getMaxAge();
         $currentSessions = $member->LoginSessions()->filterAny([
             'Persistent' => 1,
             'LastAccessed:GreaterThan' => date('Y-m-d H:i:s', $maxAge)
         ]);
         return $currentSessions;
     }
 
+    /**
+     * Get the max age for all valid sessions
+     * @return string
```

Comment:
```suggestion
```
Please don't include the return type in the phpdoc - there's a chance it could get desynced from the actual PHP typehint with changes down the line.